widgy/compat.py

```diff
@@ -0,0 +1,20 @@
+import django
+
+
+def check_if_field_name_exists_on_cls(field_name, cls):
+    """
+    Check if a field is already present on a model in a `safe` manner that does
+    not trigger or violate application loading.
+    """
+    if django.get_version() < "1.7":
```

Comment:
@pipermerriam I edited my answer for clarification. You should be able to figure out the case it doesn't work next time ;) .